reactnative/RNPlayground/.env.template

```diff
@@ -0,0 +1,8 @@
+# This file contains keys and properties that should not be checked in to Git.
+# To use it rename this file to just `.env` and set properties you need.
+# The values are available in code via `process.env` map, for example `process.env.UNSPLASH_API_KEY`.
+# For changes to be visible you may need to restart Metro clearing its cache.
+# For more information on how it's set up see https://ignitecookbook.com/docs/recipes/EnvironmentVariables
+# More general information on security in RN https://reactnative.dev/docs/security#storing-sensitive-info
+
+UNSPLASH_API_KEY=your_key_here
```

Comment:
Do we need to update this key or remvoe this file completely?

👀  
> This file contains keys and properties that should not be checked in to Git.

Reply:
this is just an example file, to be used it would need to be renamed to `.env` at which point it would be ignored by git